src/libopensc/ctx.c

```diff
@@ -146,7 +146,9 @@ static const struct _sc_driver_entry internal_card_drivers[] = {
 #endif
 	{ "openpgp",	(void *(*)(void)) sc_get_openpgp_driver },
 	{ "jpki",	(void *(*)(void)) sc_get_jpki_driver },
+#ifdef ENABLE_OPENPACE
```

Comment:
npa should be available without OpenPACE, the crypto may be done inside the card reader...